e2e/tests/gatekeeperMakesAllocationDecision_test.js

```diff
@@ -0,0 +1,54 @@
+const config = require('../config.js');
```

Comment:
this test should belong in gatekeepingAdministersCaseAfterGatekeeping, not it's own test file

Reply:
discussed with you